service_info/templates/cms/base.html

```diff
@@ -27,7 +27,7 @@
                 <div id=menu-container class="menu-closed">
                     <button id=menu-toggle class="btn menu">{% trans "MENU" %}</button>
                     <ul id="menu" class="menu">
-                        {% show_menu 0 0 0 100 "cms/includes/menu.html" %}
+                        {% show_menu 0 1 0 100 "cms/includes/menu.html" %}
```

Comment:
If the third arg to show_menu is 1, we can see children even if they're not a child of the current node.  So this works better for me: ``{% show_menu 0 1 1 100 "cms/includes/menu.html" %}``

Perhaps the fourth arg should be 1, to not show any additional descendants of the active node?  I don't think your code is displaying any additional descendants even when they are there, so we may as well not query for them????  (not sure about this)  I originally thought it should be 0 (0 "extra") but that seems to override the depth and I don't see the child of an active node.